js/app.js

```diff
@@ -0,0 +1,17 @@
+/*global window, console, document, Image
+*/
+$(function () {
+  'use strict';
```

Comment:
`'use strict'` always at the top of file, it's information for the js parser, so should do it in the whole file ;)